eZ/Bundle/EzPublishRestBundle/Tests/Functional/TestCase.php

```diff
@@ -8,12 +8,16 @@
  */
 namespace eZ\Bundle\EzPublishRestBundle\Tests\Functional;
 
+use Buzz\Browser;
+use Buzz\Client\BuzzClientInterface;
 use Buzz\Client\Curl;
+use Nyholm\Psr7\Factory\Psr17Factory;
```

Comment:
Would it make sense to use it as `RequestFactory`? `Psr17Factory` is very enigmatic (at least for me).

Reply:
Or we could use PSR name and call it HTTPFactory

Reply:
Good idea! :tada: Fixed and squashed into e046d18

Reply:
> Or we could use PSR name and call it HTTPFactory

About that name I'm not so sure. It doesn't specify if this is for Request or Response. HTTP Factory is a set of factories implementing either RequestFactoryInterface or ResponseFactoryInterface. So naming choosen by @mnocon seems more clear (and more aligned with CI failure message.

Side stuff... it is actually ResponseFactory, my bad :)

Reply:
On second thought... I'm taking back what I've said above. Seems like Psr17Factory implements both Request and Response, so changed as @ViniTou suggested (362d63d). Naming... :scream: